### ISSUE TITLE
Wichtig: Fehler Zeitbuchungen

### ISSUE THREAD

**Issue by Christian-H** [yakamara/redaxo_prozer#48]
Der Fehler tritt anscheinend auf, wenn:
- Eine Person, eine andere auf zu einem Projekttermin einläd, in welchen die Person nicht ist
- Diese Person, den Termin bestätigt den Termin, obwohl sie nicht zugelassen ist.  (das ist ok)
- Dann wird dieser Termin in die DB als Job unter dem Projekt geschrieben. (hier ist der Fehler)

Effekt: -> Die Person kann diesen Job nicht sehen und editieren.

auch komisch: Irgendwie ist der Job sogar meistens zwei oder mehrmals angelegt, da die Leute glauben, es war ein Fehler und drücken nochmal "Job kopieren".

**Comment by frag-seb** [comment]
Ah verteh, also User die nicht im Projekt sind, aber eine Einladung zum Termin bekommen sollen diesen sehen. Wenn der Termin dann Kopiert wird sollen sie die Jobs die zu Ihnen gehören, sehen und bearbeiten können.

Eigentlich ist es inkonsistent, zusagen das jemand nichts mit dem Projekt zu tun hat, und dann ihm einzelne Termine/Jobs zuzuteilen. 

Für Personen die nicht  von Anfang an Bord sind, hört sich das seltsam an. 
Ist aber gewünscht?

**Comment by Christian-H** [comment]
Beste Frage :-)
Also ja, an sich es schon inkonsistent, dass ich eine Person zu einem Projekt termin einladen kann, die noch gar nciht im Projekt ist :-)
Mein Vorschlag wäre es, wenn ein Hinweis erscheint, dass man gerade eine Person einem Termin hinzufügen will, wozu sie gar nicht zugelassen ist. Noch besser wäre dieFrage, ob die Person, dem Projekt mit dem Recht Termin zugeordnet werden soll, wenn der Einalder auch Projektverwalter dieses Projektes ist. 
Verstanden, was ich meine?

**Comment by frag-seb** [comment]
Persönlich finde ich es nicht gut das was geschieht, von dem der User nichts mitbekommt und wo er nicht eine Bewusst Aktion ausgeführt hat. (Wenn du ein Auto kaufst, bekommst du ja nicht still und leise eine Garage dazu) verstehst du was ich meine?
Mein Vorschlage wäre eine Fehlermeldung, mit dem Hinweis das man diesem Teilnehmer bitte noch Rechte zuweisen möge und einem Verwalter einen Möglichkeit einräumt (rechte Seite „Content“), den Teilnehmer in das Projekt zutun und die Rechte zu vergeben.

**Comment by Christian-H** [comment]
Ja, der Vorschlag ist ja ähnlich, aber besser als meiner, wenn ich es richtig verstehe. Ich verstehe nur nicht, was du mit rechter Seite "Content" meinst? Würdest du dann den Kalender dafür überschreiben oder wie kann ich mir das vorstellen?

**Comment by frag-seb** [comment]
Ja, hab ich mir so gedacht, wir können auch eine "popup" (kein echtes) öffnen oder sowas..

Bin für Ideen offen :D
